### ISSUE TITLE
`get_all_schema()` error

### ISSUE THREAD

**Issue by sandreae** [p2panda/aquadoggo#145]
Currently if any `schema_definition` exists in the db and it's `schema_field_definitions` are missing, this method returns an error. It should just not return this schema.

**Comment by cafca** [comment]
I couldn't replicate this issue - has this been solved already maybe?

**Comment by sandreae** [comment]
Yes! I believe it was this which fixed it: https://github.com/p2panda/aquadoggo/pull/192
